volatility/framework/plugins/linux/lsmod.py

```diff
@@ -44,6 +47,9 @@ def list_modules(cls, context: interfaces.context.ContextInterface, layer_name:
 
         vmlinux = contexts.Module(context, vmlinux_symbols, layer_name, 0)
 
+        # this will cause an exception if module support is not compiled into the kernel
```

Comment:
If the line below is there specifically to trip the exception early, this comment should say it, otherwise it seems unnecessary (because the code will either trip or not during the course of its run?)

Reply:
good call. check again :)